types/integration/fsc.ts

```diff
@@ -44,8 +47,45 @@ export type FscClass = {
         totalBookable: number;
         reservedForDropin: number;
         leftToBook: number;
-        leftToBookIncDropin: number; // [sic]
+        leftToBookIncDropin: number; // also known as leftToBookIncludingDropIn
         hasWaitingList: boolean;
         inWaitingList: number;
     };
+}
+
+export interface FscActivityDetailResponse extends FscResponse {
+    data: FscActivityDetail;
+}
+
+export type FscActivityDetail = {
+    id: number;
+    name: string;
+    productLabels: {
+        id: number;
+        name: string;
+    }[];
+    businessUnits: {
+        id: number;
+        name: string;
+        location: string;
+        companyNameForInvoice: string;
+    }[];
+    description: string;
+    assets?: {
+        reference: string;
+        type: string;
+        contentType: string;
+        contentUrl: string;
+        imageWidth: number;
+        imageHeight: number;
+        focalPointX: number;
+        focalPointY: number;
+    }[];
+    participantsMustPay: boolean;
 };
+
+export type DetailedFscWeekSchedule = DetailedFscClass[];
+export interface DetailedFscClass extends FscClass {
```

Comment:
Any particular reason for using interface here?

Reply:
Mostly just for the purpose of DRY. Can omit it if you like, but I did not want to repeat the details of type 😛

Reply:
Yea, completely agree, it was more a question of interface extension vs type intersection

Reply:
Ah, no reason really, I am just used to using it. I have no preference on the matter, so I changed it to type intersection. (since it seemed like you prefer that)